app/src/test/java/com/kickstarter/libs/KoalaTest.kt

```diff
@@ -237,6 +245,7 @@ class KoalaTest : KSRobolectricTestCase() {
         assertEquals("portrait", expectedProperties["device_orientation"])
         assertEquals("uuid", expectedProperties["distinct_id"])
         assertEquals("unavailable", expectedProperties["google_play_services"])
+        assertEquals(false, expectedProperties["is_vo_on"])
```

Comment:
would it be possible to replicate an environment where the `AccessibilityManager` is in a state where `is_vo_on` is true? 

this is mostly to make sure we're testing under as close to real conditions.

Reply:
so since these are unit tests, not instrumented tests, i'd say no (learn more about instrumented tests here! https://source.android.com/compatibility/tests/development/instrumentation). the only think I could do would be to override `MockKoalaClient` and return `true`. but i'll be pairing with Infrastructure next week to actually run the tests on a device in Circle CI 😱